release notes/v0.44.0.md

```diff
@@ -0,0 +1,224 @@
+k6 v0.44.0 is here 🎉! This release includes:
+
+- A new experimental webcrypto module.
+- A sampling option for the tracing module.
+- Support for z/OS Unix.
```

Comment:
Let not advertise this as we in practice do not support it if for no other reason because `afero` does not with the version we are using currently
```suggestion
```